src/index.tsx

```diff
@@ -22,7 +22,12 @@ root.render(
   // of the HTML which is later parsed by PagedJS
   // <React.StrictMode>
   <React.Suspense fallback={strings.LOADING}>
-    <Router>
+    <Router
+      future={{
+        // v7_relativeSplatPath: true,
```

Comment:
Why is this commented?

Reply:
That's the other necessary piece needed to be updated before upgrading to react-router v7. When you add that future flag, you're saying that you want to use the v7 behavior instead of v6, which causes breaking changes. I left it in to help us remember what else is needed, but will work on the changes necessary for that flag in a future PR.